examples/webgl_lights_spotlight.html

```diff
@@ -116,11 +116,11 @@
 
 				//
 
-				material = new THREE.MeshPhongMaterial( { color: 0x4080ff, dithering: true } );
+				let material2 = new THREE.MeshPhongMaterial( { color: 0x4080ff, dithering: true } );
```

Comment:
Shouldn't it be `const`?

Reply:
Fixed. 5ae10928d04adc176d17e57f3782fad7eb8f7a94